README.md

```diff
@@ -1,2 +1,86 @@
-# yoti-node-sdk-sandbox
-Yoti Node SDK Sandbox
+# Yoti NodeJS Sandbox SDK
+
+Welcome to the Yoti NodeJS Sandbox. This repo contains the tools you need to test your NodeJS back-end integration.
+
+## Requirements
+
+### Node version
+Please refer to [Travis](https://travis-ci.com/getyoti/yoti-node-sdk-sandbox) to see all compatible Node versions.
+
+## Installing the Sandbox
+
+To import the Yoti SDK inside your project, you can use your favourite dependency management system.
+If you are using NPM, you can use the following command to set the Yoti Sandbox SDK as a dependency:
+
+```shell
+npm install @getyoti/sdk-sandbox
+```
+
+## Configuration
+
+* `CLIENT_SDK_ID` is the SDK identifier generated by Yoti Hub in the Key tab when you create your app.
+
+* `/path/to/your-pem-file.pem` is the path to the application pem file. It can be downloaded only once from the Keys tab in your Yoti Hub.
+
+Please do not open the pem file as this might corrupt the key and you will need to create a new application.
+
+## Profile Token Creation
+
+```javascript
+const fs = require('fs');
+
+const {
+  SandboxProfileClientBuilder,
+  SandboxAgeVerificationBuilder,
+  TokenRequestBuilder,
+} = require('@getyoti/sdk-sandbox');
+
+const yoti = require('yoti');
+
+const CLIENT_SDK_ID = 'CLIENT_SDK_ID';
```

Comment:
Do you think we should call this "Sandbox Client SDK ID", so that it's clear that both the call to the sandbox service, and the consequent call to GetActivityDetails should be used with the SandboxClientSdkId, and not their usual ClientSdkId? 
```suggestion
const SANDBOX_CLIENT_SDK_ID = 'CLIENT_SDK_ID';
```

Reply:
Changed in d7ab6d8